core/templates/dev/head/exploration_editor/settings_tab/SettingsTab.js

```diff
@@ -40,6 +40,9 @@ oppia.controller('SettingsTab', [
       EXPLORATION_TITLE_INPUT_FOCUS_LABEL);
 
     $scope.CATEGORY_LIST_FOR_SELECT2 = [];
+    //Empty category added to force dropdown to scroll top
```

Comment:
(Or rather, add a commit to the existing PR. Don't worry about making too many commits; we generally squash them before merge anyway, so it's totally fine to be incremental.)